docs/templates/config.rst.j2

```diff
@@ -103,25 +103,43 @@ you can use the command line utility mentioned above (`ansible-config`) to brows
 {{config_option}}
 {{ '-' * config_len }}
 
-{% if config['description'] and config['description'] != [''] %}
-{% if config['description'] != ['TODO: write it'] %}
+{% if config.get('description', '') not in ('', ['TODO: write it']) %}
 :Description: {{' '.join(config['description'])}}
 {% endif %}
-{% endif %}
 {% if config['type'] %}
 :Type: {{config['type']}}
 {% endif %}
+{% if 'default' in config %}
 :Default: {{config['default']}}
+{% endif %}
+{% if config.get('choices', False) %}
+:Choices:
+{% if config['choices'] is mapping %}
```

Comment:
Why add choices to be able to be a dict?

Reply:
cause it seemed a much better way to document them vs having them copied again in description or elsewhere

Reply:
I guess just wondering why it's better than a list. Aren't lists already supported?

Reply:
no, they are currently 'ignored', this adds support for both as 'choices'